drivers/i2c/i2c_ite_it8xxx2.c

```diff
@@ -534,11 +532,13 @@ static int i2c_it8xxx2_transfer(const struct device *dev, struct i2c_msg *msgs,
 
 		if (msgs->flags & I2C_MSG_START) {
 			data->i2ccs = I2C_CH_NORMAL;
-			/* enable i2c interrupt */
-			irq_enable(config->i2c_irq_base);
 		}
+
 		/* Start transaction */
-		i2c_transaction(dev);
+		if (i2c_transaction(dev)) {
```

Comment:
Same for this file - document the return value for `i2c_transaction()` and `i2c_tran_read/write()`

Reply:
Done.